README.md

```diff
@@ -88,7 +88,7 @@ A `Dockerfile` is included, you can build and run it with:
 
 ```bash
 docker build -t camo .
-docker run --env CAMO_KEY=YOUR_KEY -t camo
```

Comment:
Please help me I don't even know what all this stuff is all I know is my ex boy friend hacked my phone and laptop .
Please can someone explain to me what all this is